CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ You will need to first install [Homebrew](https://brew.sh).
 - `brew install libsndfile lame` (required for id3 tags and waveforms)
 - `brew install vips` (required for processing images)
 - `brew install yarn` (required for building assets)
+- `brew install mysql` (required for local database)
+  - before running rake to generate the data, you'll also need to start the mysql server, as well as have the root user with password 'root' and a currently logged in user with no password.
+  - you can find currently logged in user by doing: `stat -f '%Su' /dev/console` will refer to this as currentUser
+  - `mysql.server start`
+  - `mysql -u root -h localhost` to open root mysql prompt.  In the prompt type the following lines, replacing currentUser with what you figured it out to be above
+  - `SET PASSWORD FOR root@localhost='root';`
+  - `CREATE USER 'currentUser'@'localhost' IDENTIFIED BY '';`
+  - `GRANT ALL PRIVILEGES ON * . * TO 'currentUser'@'localhost';`
+
```

Comment:
Good idea to add info on mysql.The instructions after `myqsl.server start`  shouldn't be  necessary, as mysql ships with a root user and no password, which matches the setup that alonetone expects. So I believe if you re-did the setup from scratch and didn't want to use a custom user, it should work out of the box.

Reply:
Cool!  I do think the root password may need to be updated.  It looks like it was changed to 'root' because this is what github actions expects, and that is different than the brew default of user 'root' not having password.  I removed the other lines though.  Happy to change if I'm not understanding something.